src/components/Table/Table.tsx

```diff
@@ -113,6 +113,15 @@ const Table = ({
       onChange: onRowSelectionChange,
     })
 
+  const loadingRows: Array<DataType> = useMemo(() => {
+    const rowsArray = []
+    // eslint-disable-next-line unicorn/new-for-builtins
+    for (let index = 0; index < initialPageSize; index++) {
+      rowsArray.push({ id: `${index}` })
+    }
+    return rowsArray
+  }, [initialPageSize])
+
```

Comment:
C: It could be simpler like this and without eslint errors
```suggestion
  const loadingRows: Array<DataType> = useMemo(
    () =>
      Array.from({ length: initialPageSize })
        .fill("")
        .map((_, index) => ({ id: `${index}` })),
    [initialPageSize]
  )
```